omnibus/cookbooks/omnibus-supermarket/recipes/rails.rb

```diff
@@ -30,32 +30,36 @@
   end
 end
 
-template "#{node['supermarket']['var_directory']}/etc/unicorn.rb" do
+template 'unicorn.rb' do
+  path "#{node['supermarket']['var_directory']}/etc/unicorn.rb"
   source 'unicorn.rb.erb'
   cookbook 'unicorn'
   owner node['supermarket']['user']
   group node['supermarket']['group']
   mode '0600'
   variables(node['supermarket']['unicorn'].to_hash)
-  notifies :restart, 'runit_service[rails]'
 end
 
-template "#{node['supermarket']['nginx']['directory']}/sites-enabled/rails" do
+template "rails.nginx.conf" do
+  path "#{node['supermarket']['nginx']['directory']}/sites-enabled/rails"
   source 'rails.nginx.conf.erb'
   owner node['supermarket']['user']
   group node['supermarket']['group']
   mode '0600'
   variables(nginx: node['supermarket']['nginx'],
             rails: node['supermarket']['rails'],
             fqdn: node['supermarket']['fqdn'],
+            fips_enabled: node['supermarket']['fips_enabled'],
             ssl: node['supermarket']['ssl'],
             app_directory: node['supermarket']['app_directory'])
-  notifies :reload, 'runit_service[nginx]' if node['supermarket']['nginx']['enable']
```

Comment:
Why the general change to shift to `subscribes` for the service to these files, instead of `notifies` here?

Reply:
answer: 'cause it's a proper resource now

Reply:
In getting chefspec to pass again for this cookbook, it seems chefspec doesn't handle notifies targeting resources that are declared later. Also, these notifies were targeting `runit_service` resources that are declared inside the `component_runit_service` resource, which seems dirty. That's among the reasons why I [added these actions](https://github.com/chef-cookbooks/enterprise-chef-common/pull/67/files#diff-ee7d560240df9baed002da7d844d9576R108) to the `component_runit_service` custom resource in chef-cookbooks/enterprise-chef-common#67.